### ISSUE TITLE
Can you support scaling of images?

### ISSUE THREAD

**Issue by alexellis** [mandolyte/mdtopdf#12]
Can you support scaling of images in this project? They are coming out at 100% which can be disproportional for the page, in the context of an eBook or white-paper this would be particularly useful.

**Comment by mandolyte** [comment]
@alexellis I looked for ways to specify in the markdown source the scaling. Found [this](https://www.xaprb.com/blog/how-to-style-images-with-markdown). There doesn't seem to be any standards here. I also checked [Obsidian](https://obsidian.md/). Turns out it has its own way of scaling and since it supports single Markdown to PDF, it might work for your use case.

If still interested, my next step would be to adopt some "standard" that I can follow. I also need to check the `gofpdf` package to be sure it supports scaling in the PDF output.

Thanks for contacting me!
